.github/workflows/manual_release.yaml

```diff
@@ -0,0 +1,55 @@
+name: Update CITATION.cff version and create a new release
+on:
+  workflow_dispatch:
+    inputs:
+      version:
+        description: 'New version'
+        required: true
+
+jobs:
+  update-version-and-release:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v4
+        with:
+          fetch-depth: 0
+
+      - name: Update version
+        run: |
+          sed -i "s/version: .*/version: ${{ github.event.inputs.version }}/g" CITATION.cff
+
+      - name: Commit changes to a new branch
+        run: |
+          const { version } = github.event.inputs;
+          git config --local user.email "action@github.com"
+          git config --local user.name "GitHub Action"
+          git checkout -b update-version-${version}
+          git commit -am "Update CITATION.cff version to ${version}"
+          git push origin update-version-${version}
+
+      - name: Create pull request
+        uses: actions/github-script@v6
+        with:
+          script: |
+            const { version } = github.event.inputs;
+            github.pulls.create({
+              owner: context.repo.owner,
+              repo: context.repo.repo,
+              title: `Update CITATION.cff version to ${version}`,
+              head: `update-version-${version}`,
+              base: `main`,
+              body: `This PR updates the CITATION.cff version to ${version}`
+            })
+
+      - name: Create release
```

Comment:
In this form, the PR gets created, but the release does not depend on the PR being merged, so the publish pipeline would fail because of version mismatch.

We either need to separate actions or allow GitHub actions bot to bypass branch protection.

Reply:
I think we can remove this file and add the automation in another PR